sandbox/src/mocks/cancelReferral/responses/ExampleCancelledReferralWithAppoinment.json

```diff
@@ -48,6 +48,31 @@
                             "system": "https://fhir.nhs.uk/Id/ods-organization-code",
                             "value": "R69"
                         }
+                    },
+                    {
+                        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralState-1",
+                        "extension": [
+                            {
+                                "url": "state",
+                                "valueCoding": {
+                                    "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1",
+                                    "code": "CANCELLED_REFERRAL",
+                                    "display": "Cancelled Referral"
+                                }
+                            },
+                            {
+                                "url": "reason",
+                                "valueCoding": {
+                                    "system": "_baseurl_/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON",
```

Comment:
I think the cancellation reason here should be an `REQUEST-CANCELLATION-REASON` rather than an `APPOINTMENT-CANCELLATION-REASON` as the `state` relates to the `ReferralRequest`. Would the API implementation also include a `REQUEST-CANCELLATION-REASON` in this scenario?

Reply:
Yes API implemented REQUEST-CANCELLATION-REASON. Corrected.